src/applications/vaos/services/healthcare-service/index.js

```diff
@@ -0,0 +1,37 @@
+import { getAvailableClinics } from '../../api';
+import { transformAvailableClinics } from './transformers';
+import { mapToFHIRErrors } from '../../utils/fhir';
+import { getLocation } from '../location';
+
+/*
+ * This is used to parse the fake FHIR ids we create for organizations
+ */
+function parseId(id) {
+  return id.replace('var', '');
+}
+
+export async function getAvailableHealthcareServices(
```

Comment:
Should have a jsdoc comment here. Also, we've been passing parameters in as an object for these exported functions that make api calls. You can look at the other services as an example.

Reply:
What is the purpose of passing the parameters as an object?

Reply:
We expect to have some parameter name changes or new params for these services as we update and adjust things (since terminology is changing so much), so it's a little more robust to have an object where we can effectively have named parameters.

Reply:
Done.